_data/meltano/utilities/superset/apache.yml

```diff
@@ -0,0 +1,32 @@
+name: superset
+namespace: superset
+label: Apache Superset
+description: A modern, enterprise-ready business intelligence web application.
+docs: https://docs.meltano.com/guide/orchestration
+repo: https://github.com/apache/superset
+variant: apache
+pip_url: apache-superset==1.5.0 markupsafe==2.0.1
+requires:
+  files:
+  - name: files-superset
+    variant: meltano
+settings:
+- name: home
+  value: $MELTANO_PROJECT_ROOT/.meltano/run/superset
+  description: The home directory where Superset will store data.
+- name: additional_dependencies
+  description: An space separated list of python dependencies to include as part of startup.
+    A list of database driver dependencies can be found here https://superset.apache.org/docs/databases/installing-database-drivers
+- name: load_examples
+  kind: boolean
+  value: false
+  description: A boolean whether to load Superset's example datasets and dashboards. Warning, this takes a few mins to load.
+- name: config_path
+  value: $MELTANO_PROJECT_ROOT/analyze/superset/superset_config.py
```

Comment:
dbt lives in `/transform`, and Airflow DAGs live in `/orchestrate`, without `/dbt` or `/airflow` added into the path. What do you think about being consistent and using `/analyze` here (and above, and in the file bundle) without `/superset`? 

Or would it be better to change the others to use the plugin name explicitly in the path as well, in situations where you'd have multiple plugins of a type?

@tayloramurphy Wdyt?

Reply:
@DouweM yeah I noticed that too, I think its better to namespace them. Probably for other cases too but especially for someone exploring plugins locally they could have multiple analyze plugins installed, maybe lightdash and superset to compare, and they'd get this shared directory with all files lumped together. Same with dbt, like they could have multiple warehouses and include dbt-redshift and dbt-snowflake.

Reply:
@DouweM I'm with @pnadolny13 on this one. Namespacing the directories will likely be preferred going forward. Especially if we want people to maintain everything in one repo, it's not a good assumption that they're going to have only a single tool for each top-level directory.